systemtests/src/test/java/io/enmasse/systemtest/marathon/MarathonTestBase.java

```diff
@@ -310,9 +310,17 @@ void doTestCreateDeleteAddressesViaAgentLong(AddressSpace addressSpace) throws E
         ArrayList<Destination> addresses = generateQueueTopicList("via-web", IntStream.range(0, addressCount));
 
         runTestInLoop(30, () -> {
-            consoleWebPage.createAddressesWebConsole(addresses.toArray(new Destination[0]));
-            consoleWebPage.deleteAddressesWebConsole(addresses.toArray(new Destination[0]));
-            Thread.sleep(5000);
+            try {
+                consoleWebPage.createAddressesWebConsole(addresses.toArray(new Destination[0]));
+                consoleWebPage.deleteAddressesWebConsole(addresses.toArray(new Destination[0]));
+                Thread.sleep(5000);
+                selenium.saveScreenShots("testClass", "testName");
```

Comment:
Sorry for confusing, with "testClass" and "testName" I mean a real testclass and testname